sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/AutoCompleteExample.java

```diff
@@ -46,11 +43,8 @@ private static void autoCompleteWithOneTermContext(SearchClient searchClient) {
         AutocompleteOptions params = new AutocompleteOptions().setAutocompleteMode(
             AutocompleteMode.ONE_TERM_WITH_CONTEXT);
 
-        RequestOptions requestOptions = new RequestOptions()
-            .setClientRequestId(UUID.randomUUID());
-
         PagedIterableBase<AutocompleteItem, AutocompletePagedResponse> results = searchClient.autocomplete("coffee m",
```

Comment:
Could you show the example of how to set request id?

Reply:
I'll add an example of how to add per-request `x-ms-client-request-id` in the next commit.